lib/main_common.pm

```diff
@@ -157,6 +158,11 @@ sub is_bridged_networking {
     return $ret;
 }
 
+sub replace_opensuse_repos_tests {
```

Comment:
why you put it in main_common ? because it will be used ONLY in opensuse it should be in opensuse/main.pm